src/Components/APICalls.js

```diff
@@ -117,23 +107,34 @@ export function useRecommendations(viewHistory) {
   return useQuery(
     ["Recommendations", viewHistory],
     async () => {
-      let response = await fetch(
-        `https://api-jet-lfoguxrv7q-uw.a.run.app/recommend`,
-        {
-          method: "POST",
-          mode: "cors",
-          headers: {
-            "Content-Type": "application/json",
-          },
-          body: JSON.stringify(viewHistory),
-        }
-      );
+      let response = await fetch(`${apiUrl}/recommend`, {
+        method: "POST",
+        mode: "cors",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(viewHistory),
+      });
       await handleErrors(response);
       let responseJson = await response.json();
       let temp = [];
       responseJson.items.map((item, index) => temp.push(item.anime));
+      console.log(temp);
       return temp;
     },
     { staleTime: fiveMinutesMs, keepPreviousData: true }
   );
 }
+
+export async function getRandomAnimeListing(randomPage, randomItem) {
```

Comment:
Nice!  This looks great, and I love getting some of the logic of getting the random shelf out from `Home`.